server/.env.SAMPLE

```diff
@@ -22,4 +22,7 @@ COMPANY_NAME = ""
 
 # Key for Open Weather API
 WEATHER_API_KEY = ""
-WEATHER_API_CITY_ID = ""
\ No newline at end of file
+WEATHER_API_CITY_ID = ""
+
+#Trivia config
+TRIVIA_API_URL = "http://numbersapi.com"
```

Comment:
I'd prefer to put this in constants or default config. The whole implementation or the trivia widget is coupled to this site so there is no need to make this URL configurable.